README.md

```diff
@@ -1,280 +1,231 @@
+```
+[foreign_chain]
+rpc_url = "http://<goerli node IP address>:8545"
+token_contract_address = "0x54B06531214AD41DE9d771c10C0030d048d0cC67"
+bridge_contract_address = "0x2171Dd4d4F6ca30FeEA8a27b96257A67f371d87A"
+event_fetch_start_block_number = 1321331
+
+[home_chain]
+rpc_url = "http://<laika node IP address>:8545"
+bridge_contract_address = "0x854dF872BB8bfECafFB1077FCfd7aa0B7C838A60"
+event_fetch_start_block_number = 4153205
+
+[validator_private_key]
+keystore_path = "/config/keys/Trustlines/<keystore_filename>.json
+keystore_password_path = "/config/pass.pwd"
 ```
 
-Otherwise, you need to adjust some parameters. You need to provide the path to your private key in "--keys-path=[path]", the path to your password protecting this key in "--password=[path]" as well as your address in "--author=[address]" and "--engine-signer=[address]"
+Note that they keystore path is not an actual path on the host machine, but rather in the bridge container we will create next. The container will have to connect to both of the docker networks and access the config directory. Therefore, the command looks like this:
 
 ```sh
-parity --chain trustlines-spec.json -d validator_node --auto-update=none --no-download --keys-path=[path/to/keys] --password=[path/to/password] --max-peers=100 --author=[address] --engine-signer=[address] --force-sealing --reseal-on-txs=none --min-gas-price="1" --tx-queue-per-sender=100
+$ docker run -d --name bridge-client --network network-laika --network network-ethereum -v $(pwd)/trustlines/config:/config -v $(pwd)/trustlines/bridge-config.toml:/config/bridge-config.toml trustlines/bridge-next:master -c /config/bridge-config.toml
 ```
 
----
+### Setup Without Docker
 
-## Backups
+We refer to the documentation of the individual components:
 
-Please make sure to backup your private key and associated password! Depending on your setup, the key can be found in general in _Parity's_
-home directory (per default `~/.local/share/io.parity.ethereum`) under `keys/Trustlines/`. In case of using the docker image, it can be found
-in the volume bound to `/config` relative to [these instructions](#participant). When working with the quickstart script, the key is placed in
-`./trustlines/config/keys/Trustlines` relative to where you have run the script.
+- [Parity](https://wiki.parity.io/Parity-Ethereum)
+- [Netstats](https://github.com/trustlines-protocol/ethstats-client)
+- [Monitor](https://github.com/trustlines-protocol/tlbc-monitor)
+- [Bridge](https://github.com/trustlines-protocol/blockchain/tree/master/tools/bridge)
 
----
+For the Laika node, make sure it uses the correct [chain spec file](), that the right TCP and UDP ports are used
+(30300), and that the JSON RPC APIs `web3`, `eth`, and `net` are open.
```

Comment:
```suggestion
(30300), and that the JSON RPC APIs `web3`, `eth`, and `net` are enabled.
```
To do not mix it with open the port. Just to make it foolproof regarding security.